packages/integrations/image/README.md

```diff
@@ -169,14 +169,16 @@ Set to an empty string (`alt=""`) if the image is not a key part of the content
 
 <p>
 
-**Type:** `'avif' | 'jpeg' | 'png' | 'webp'`<br>
+**Type:** `'avif' | 'jpeg' | 'jpg' | 'png' | 'svg' | 'webp'`<br>
 **Default:** `undefined`
 </p>
 
 The output format to be used in the optimized image. The original image format will be used if `format` is not provided.
 
 This property is required for remote images only, because the original format cannot be inferred.
 
+The `svg` format is only valid when the original image is already in SVG format. The image itself won't be processed but the resulting `<img />` will have the right attributes.
```

Comment:
I'm tempted to make this a note, since this format works differently from all the others. When using the `svg` option:
- the original image must be in SVG format already
- the image integration cannot transform a non-SVG image into an SVG
- remote SVG images are not "processed" (does this mean optimized? or does this mean "changed to this format"? Processed could mean different things.) If the image is not optimized by the integration, that's another difference when setting the type to `svg`.

This seems like enough to call out as something that really is "not like the others" and might even by helpful to mention when/why you would choose to use the image integration for an svg, instead of choosing just an `<img>` tag. It's not clear to me from this that the image integration does much for you here.  :smile:

Reply:
Thank you @sarah11918 for looking into this. :)

Yes, by "processed" I meant transformed/optimized.

The image integration not only transforms the images but also reads the width and height of the image to put them into the `width` and `height` attributes of the `<img>` tag, which optimizes the cumulative layout shift (CLS). 

Transformation with SVG output is not possible but CLS optimization is, so the image integration is still useful for SVG images.

Moreover, making `<Image />` accept all image formats that work with `<img />` allows the developer to not worry about SVG's specific case when using a bunch of images of various formats.

I hope I was clearer. I can add more details to the README if necessary.

Reply:
I updated the README. Hope it's better now :)

Reply:
> The image integration not only transforms the images but also reads the width and height of the image to put them into the `width` and `height` attributes of the `<img>` tag, which optimizes the cumulative layout shift (CLS).

@ggounot You are absolutely right; I am reading more and more about how specifying `width` and `height` attributes on `<img>` tags is so important now.

(I was also trying to do this on a site built with Astro, but as all images (binary and SVGs) are already optimized, I was only trying to automatically extract the width and height of each image and add them to the `<img>` tag. BTW, what are you using to extract the SVG width and height in your code?)